### PR TITLE
Added geolocation

### PR DESCRIPTION
**Pull request by rafisbytes**
I added `expo-location` to find the geolocation, and it sets the location of in map on the home screen to the geolocation.

### CODE REVIEW

File: App.js [hunk 1]
```diff
@@ -68,6 +70,45 @@ const location = {
 };
 
 function HomeScreen({ navigation }) {
+  const [geoLocation, setGeoLocation] = useState(null);
+  const [errorLocationMsg, setErrorLocationMsg] = useState(null);
+
+  function convertToMapLocation(geoLoc) {
+    if (geoLoc) {
+      console.log(geoLoc);
```
Comment (nikfrank) [inline App.js]: Hahahahahha i caught you committing a log!

File: App.js [hunk 2]
```diff
@@ -68,6 +70,45 @@ const location = {
...
+  let myLocation = null;
+  if (errorLocationMsg) {
+    alert(errorLocationMsg);
+  } else if (geoLocation) {
+    myLocation = {
+      latitude: geoLocation.latitude,
+      longitude: geoLocation.longitude,
+      latitudeDelta: 0.009,
+      longitudeDelta: 0.009,
+    };
```
Comment (nikfrank) [inline App.js]: Isn't this the same code in the convert function that makes the geolocation variable that's being used in the map component?

It looks like you're halfway done refactoring (and you skipped the best part - deleting code!)

**Review by nikfrank — COMMENTED** [review]
Good job withe use effect and async function

And good shabbas from new Orleans!

**Comment by rafisbytes** [comment]
I fixed it.

**Pull request merged**